modules/Story/HeaderRenderer/HeaderRenderer.module.scss

```diff
@@ -0,0 +1,12 @@
+.renderer {
+    :global {
+        /* stylelint-disable selector-class-pattern */
```

Comment:
I noticed in other PRs you had added `, max-nesting-depth ` to the `stylelint-disable` - do you need that here as well?

Reply:
It's not needed here because there are no nested rules.

Reply:
> All code/CSS changes make sense to me. Will leave approval to those more familiar with React.

I was more interested in the design review but appreciate your coding skills as well 😛